### PR TITLE
104 - Prompt Blueprint Documentation

### LINKED ISSUE #104: Prompt Blueprint Documentation
Right now, the Prompt Blueprint format is not documented.

We need a full page on the docs that explains what a user should expect from it.

For example, when constructing messages to inject into the placeholder, both Buck and I had trouble figuring it out.

Also understanding the response from run. Where is the ID? what should it look like? etc

### LINKED ISSUE #119: Docs for run()

### LINKED ISSUE #120: Docs for run_workflow()

### PR DESCRIPTION
**Pull request by buckyroberts**
Closes #104 
Closes #119 
Closes #120

### CODE REVIEW

File: running-requests/promptlayer-run-workflow.mdx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+<CodeGroup>
+
+```python Python
+from promptlayer import PromptLayer
+
+pl = PromptLayer(api_key="your_api_key")
+
+response = pl.run_workflow(
+    workflow_name="Your Workflow Name",
+    workflow_version_number=1
```
Comment (jzone3) [inline running-requests/promptlayer-run-workflow.mdx:19]: How come it is `prompt_version` in pl.run but it's `workflow_version_number` here. Let's make sure these are consistent.

File: running-requests/prompt-blueprints.mdx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+With this approach, you can update from one provider to another (e.g., OpenAI to Anthropic) without any code changes.
+
+## Understanding Placeholder Messages
+
+Placeholder Messages are a powerful feature that allows you to inject messages into a prompt template at runtime. By using the `placeholder` role, you can define placeholders within your prompt template that can be replaced with full messages when the prompt is executed.
+
+For more detailed information on Placeholder Messages, including how to create and use them, please refer to our dedicated [Placeholder Messages Documentation](/features/prompt-registry/placeholder-messages) page.
+
+### Running a Template with Placeholders
```
Comment (jzone3) [inline running-requests/prompt-blueprints.mdx:38]: How do I built a blueprint with functions? I think it's important that these docs are complete. Missing a lot as of now (but better than nothing obviously)

File: running-requests/promptlayer-run-workflow.mdx (hunk 1 above)
Comment (buckyroberts) [reply running-requests/promptlayer-run-workflow.mdx:19]: It was confusing to know whether `workflow_version` referred to the workflow version ID or the workflow version number. I'll make a ticket to change it to `workflow_version` in the libraries then update these docs accordingly.

File: running-requests/prompt-blueprints.mdx (hunk 2 above)
Comment (buckyroberts) [reply running-requests/prompt-blueprints.mdx:38]: You mean to include instructions on how to build a prompt template with function/tools on the UI and then show in code how to use those?

**Pull request merged**